service/service.go

```diff
@@ -177,11 +187,26 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	// Here we create the operator framework and configure resources that it can
+	// reconcile.
+	//
+	// NOTE that the order of the namespace resource is important. We have to
+	// start the namespace resource at first because all other resources are
+	// created within this namespace.
 	var operatorFramework *framework.Framework
 	{
 		frameworkConfig := framework.DefaultConfig()
 
 		frameworkConfig.Logger = config.Logger
+		frameworkConfig.Resources = []framework.Resource{
+			namespaceResource,
+
+			configMapResource,
+			deploymentResource,
+			ingressResource,
+			pvcResource,
+			serviceResource,
+		}
```

Comment:
The framework itself is now configured with the resources, not the specific operator service. Now we can use the common cache resource event handler. I think it is better that way.